miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcher.kt

```diff
@@ -11,12 +12,28 @@ import com.rakuten.tech.mobile.miniapp.storage.StorageState
 internal class MiniAppSecureStorageDispatcher(
     private val storageMaxSizeKB: Int
 ) {
-    private lateinit var bridgeExecutor: MiniAppBridgeExecutor
-    private lateinit var activity: Activity
-    private lateinit var miniAppId: String
-    private lateinit var secureStorage: MiniAppSecureStorage
-    private var cachedItems: Map<String, String>? = null
-    private var storageState: StorageState = StorageState.DEFAULT
+    @VisibleForTesting
+    internal lateinit var bridgeExecutor: MiniAppBridgeExecutor
+    @VisibleForTesting
+    internal lateinit var activity: Activity
+    @VisibleForTesting
+    internal lateinit var miniAppId: String
```

Comment:
I think, `bridgeExecutor`, `activity` and `miniAppId` fields here could be private for now 🤔